src/miragejs/routes/tags.js

```diff
@@ -0,0 +1,7 @@
+export default function tagsRoutes(server){
+    server.get("/nodes/:id/tags", (schema, request) => {
+      let id = request.params.id;
```

Comment:
Why do you get the `id` if you're returning `[]`?

Reply:
Solved, actually should be something like this: `return schema.skus.find(id)? schema.skus.find(id) : []`